gradio/utils.py

```diff
@@ -107,8 +107,10 @@ def ipython_check() -> bool:
     return is_ipython
 
 
-def is_space() -> bool:
-    return os.getenv("SYSTEM") == "spaces"
+def get_space() -> str | None:
```

Comment:
this will first check that the env variable "SYSTEM" is set to "spaces", which is the "official" indicator of whether we are in a space. I don't think it really matters but its slightly safer to first check the "system" variable first imo.